src/pages/OrganizationDetail.tsx

```diff
@@ -39,18 +46,31 @@ const OrganizationDetailPanelLoader = () => {
 	const params = useParams();
 	const { provider, organizationId = 'missing' } = params;
 	const [organization] = useOrganization(organizationId);
-
+	const [fields] = useBaseFields();
+	const [proposals] = useProposalsByOrganizationId(
+		PROPOSALS_DEFAULT_PAGE,
+		PROPOSALS_DEFAULT_COUNT,
+		organizationId,
+	);
+	const emptyProposalArray: FrontEndProposal[] = [];
+	const [proposalState, setProposalState] = useState(emptyProposalArray);
+	const emptyRecord: Record<string, string> = {};
+	const [fieldsState, setFieldsState] = useState(emptyRecord);
 	useEffect(() => {
 		if (organization === null) {
 			document.title = 'Loading... - Philanthropy Data Commons';
 		} else {
 			document.title = `${organization.name} Organization Detail - Philanthropy Data Commons`;
 		}
+		if (fields && proposals) {
+			setFieldsState(mapFieldNames(fields));
+			setProposalState(mapProposals(fields, proposals.entries));
+		}
+
 		return () => {
 			document.title = 'Philanthropy Data Commons';
 		};
-	}, [organization]);
-
+	}, [fields, proposals, organization]);
 	if (organization === null) {
```

Comment:
Add a newline here as well, to separate the rendering block from the `useEffect` block.